### PR TITLE
DR-78--Hide manage viewers button

### PR DESCRIPTION
**Pull request by RoriCremer**

### CODE REVIEW

**Review by thathert — APPROVED** [review]
LGTM

File: src/components/DatasetDetailView.jsx [hunk 1]
```diff
@@ -76,11 +77,15 @@ export class DatasetDetailView extends React.PureComponent {
...
-    const { classes, dataset, datasetPolicies, dispatch } = this.props;
+    const { classes, dataset, datasetPolicies, dispatch, userEmail } = this.props;
     const datasetReadersObj = datasetPolicies.find(policy => policy.name === 'reader'); // TODO make this an enum
     const datasetReaders = (datasetReadersObj && datasetReadersObj.members) || [];
     const datasetCustodiansObj = datasetPolicies.find(policy => policy.name === 'custodian');
     const datasetCustodians = (datasetCustodiansObj && datasetCustodiansObj.members) || [];
+    const datasetStewardsObj = datasetPolicies.find(policy => policy.name === 'steward');
+    const datasetStewards = (datasetStewardsObj && datasetStewardsObj.members) || [];
+    const allowedToAddReaders =
+      _.includes(datasetStewards, userEmail) || _.includes(datasetCustodians, userEmail);
```
Comment (thathert) [inline src/components/DatasetDetailView.jsx:88]: This won't work. We store the stewards group in the policy, not the individual steward emails. We should really be doing this by asking SAM if the user is allowed to perform the `share_policy` action on the dataset.

Comment (thathert) [reply src/components/DatasetDetailView.jsx:88]: This is fine for now. I added [this as a follow-on.](https://broadworkbench.atlassian.net/browse/DR-392)

**Review by thathert — COMMENTED** [review]
added one more detail

**Comment by ruchim** [comment]
Closing because it is now stale

**Pull request closed without merging**